models/users.lua

```diff
@@ -1,8 +1,53 @@
+local bcrypt = require ("bcrypt")
+local config   = require("lapis.config").get()
+local token = config.secret
+local trim = require("lapis.util").trim_filter
 local Model = require("lapis.db.model").Model
 local Users = Model:extend("users", {
   relations = {
     {"packages", has_many = "Packages"}
   }
 })
 
+function Users:create_user(user)
+  trim(user, {"name", "password", "admin", "email"}, nil)
+  local hash = bcrypt.digest(user.name .. user.password .. token, config.salt)
+  user.password = nil
+  local user = self:create {
+    name = user.name,
+    password = hash,
+    admin = user.admin,
+    email = user.email
+  }
+  if user then
+    return user
+  end
+  return false, {"err_create_user", {user.name}}
```

Comment:
Is the nested tables a Lapis thing?

Reply:
I believe that's being fed through to their assert_error at the end but I'll double check. That might just be a mistake.

Reply:
changed pattern a bit since I'm not making use of their custom utility. Gonna go ahead and mark these two resolved